index.html

```diff
@@ -25,8 +25,8 @@
             <div class="card-detail">
               <h3>{{activeGroup.name}}</h3>
               <div class="info pull-left">
-                <div v-if="activeGroup.motpLink" class="event-link">
-                  <a :href="activeGroup.motpLink" target="_blank">Find out more</a>
+                <div v-if="activeGroup.eventLink" class="event-link">
```

Comment:
Looks like March On's data still has some events from last year in their spreadsheet. I've asked for them to be removed, but until they are, let's not take this change.

Reply:
What if we were to put in code that handled both cases, so that it will work with whichever label is passed? This of course assumes they are never BOTH passed...
Like this:
                 <div v-if="activeGroup.eventLink" class="event-link">
                  <a :href="activeGroup.eventLink" target="_blank">Find out more</a>
                </div>
                <div v-if="activeGroup.motpLink" class="event-link">
                  <a :href="activeGroup.motpLink" target="_blank">Find out more</a>
                </div>

Reply:
Good point. It could use v-else-if to avoid the chance of seeing both.